passes/techmap/alumacc.cc

```diff
@@ -55,27 +55,27 @@ struct AlumaccWorker
 
 		RTLIL::SigSpec get_gt() {
 			if (GetSize(cached_gt) == 0)
-				cached_gt = alu_cell->module->Not(NEW_ID, alu_cell->module->Or(NEW_ID, get_lt(), get_eq()));
+				cached_gt = alu_cell->module->Not(NEW_ID, alu_cell->module->Or(NEW_ID, get_lt(), get_eq()), false, alu_cell->attributes["\\src"].decode_string());
```

Comment:
Commit 8530333 adds `get_src_attribute()` and `set_src_attribute()`. Using `get_src_attribute()` has the advantage that it does not create an empty src attribute on objects that do not have one yet. Please update your PR to use this two functions.

Reply:
Sounds good. I have updated the PR to use the new functions - please let me know if anything else is needed.